cli/.js/src/main/scala/aqua/run/RunOpts.scala

```diff
@@ -51,16 +51,20 @@ object RunOpts extends Logging {
   }
 
   def runOptsCompose[F[_]: Files: Concurrent]
-    : Opts[F[ValidatedNec[String, (Path, List[Path], FuncWithData, Option[NonEmptyList[JsonService]], List[String])]]] = {
+    : Opts[F[ValidatedNec[String, (AquaPath, List[Path], FuncWithData, Option[NonEmptyList[JsonService]], List[String])]]] = {
     (
-      AppOpts.inputOpts[F],
+      AppOpts.wrapWithOption(AppOpts.inputOpts[F]),
       AppOpts.importOpts[F],
       ArgOpts.funcWithArgsOpt[F],
       AppOpts.wrapWithOption(JsonService.jsonServiceOpt),
       AppOpts.wrapWithOption(Plugin.opt)
     ).mapN { case (inputF, importF, funcWithArgsF, jsonServiceOp, pluginsOp) =>
       for {
-        inputV <- inputF
+        inputV: ValidatedNec[String, AquaPath] <-
+          inputF.map(_.map(_.map(p => RelativePath(p)))).getOrElse {
+            println("No input specified. Default 'builtin.aqua' will be used.")
```

Comment:
I don't think this should be printed. Will confuse most of the users cuz they might not even know about `builtin.aqua`, and it already sounds like an error or warning like something is wrong.